CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Rename `LogEntry` to `SentryMessage`. Change type of `SentryEvent.Message` from `string` to `SentryMessage`.
 * Change the type of `Gpu.VendorId` from `int` to `string`.
 * Add support for envelopes.
+* Souce-link integration.
```

Comment:
We already had sourcelink but pdbs were only shipped inside the `nupkg`.
Maybe:
```suggestion
* Publishing symbols package (snupkg) to nuget.org with sourcelink
```